mne/epochs.py

```diff
@@ -842,8 +842,19 @@ def plot_psd(self, fmin=0, fmax=np.inf, proj=False, n_fft=256,
             End frequency to consider.
         proj : bool
             Apply projection.
-        n_fft : int
-            Number of points to use in Welch FFT calculations.
```

Comment:
yep - I'll update all that if people are happy with the changes

On Sun, Jan 10, 2016 at 11:12 PM, Mainak Jas <notifications@github.com>
wrote:

> In mne/epochs.py
> <https://github.com/mne-tools/mne-python/pull/2710#discussion_r49294105>:
>
> > @@ -842,8 +842,19 @@ def plot_psd(self, fmin=0, fmax=np.inf, proj=False, n_fft=256,
> >              End frequency to consider.
> >          proj : bool
> >              Apply projection.
> > -        n_fft : int
> > -            Number of points to use in Welch FFT calculations.
>
> can you add it to the whats_new section if it's a bug
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mne-tools/mne-python/pull/2710/files#r49294105>.
>